src/lib/journey/journey.store.ts

```diff
@@ -7,8 +7,11 @@ import type { JourneyStore, JourneyStoreValue, StackStore, StepTypes } from './j
 import { interpolate } from '$lib/_utilities/i18n.utilities';
 import {
   authIdTimeoutErrorCode,
+  initCheckValidation,
   shouldPopulateWithPreviousCallbacks,
-} from './_utilities/step.utilities';
+} from './stages/_utilities/step.utilities';
+import { buildCallbackMetadata, buildStepMetadata } from '$journey/_utilities/metadata.utilities';
```

Comment:
It's worth noting that I pulled the responsibility of this metadata orchestration out of the stage component and into the store. I felt that keeping this responsibility in the stage component confused the purpose of a Svelte component as compared to a Svelte store.

Stores are where network calls are made and data is constructed. Svelte components should be focused on how that data gets rendered to the screen. Any state that they do control is just local component state. 

Moving all of this into to the store cleans up the stage components and focuses them more on rendering.